python-avd/pyavd/_cv/workflows/finalize_workspace_on_cv.py

```diff
@@ -65,8 +65,31 @@ async def finalize_workspace_on_cv(workspace: CVWorkspace, cv_client: CVClient)
             workspace_id=workspace.id,
             request_id=workspace_config.request_params.request_id,
         )
+        # Form a list of known inactive existing devices
+        if inactive_devices := [f"{device.hostname} ({device.serial_number})" for device in devices if not device._streaming]:
```

Comment:
Last comment before I approve.

Do we need to also check if the device exists on CV here?

Let's be extra clear on the condition here to avoid confusion with typing (since `device._streaming` can be None):
```suggestion
        if inactive_devices := [f"{device.hostname} ({device.serial_number})" for device in devices if device._streaming is False]:
```
See if we need to check `device._exists_on_cv` as well.

Thanks

Reply:
Thanks for catching! Yes, since we are not returning an explicit list of existing devices (and use original `devices` instead) - streaming status can now be None as well and needs a more precise matching.
Fixed now.